micah-akpan/questioner-api
# #163356700 Build 'Get all Users' API endpoint

#### What does this PR do?
It adds few new API routes:
  - Get all users by admin API route
  - Get a specific user by admin route
#### Description of Task to be completed?
Build new API endpoints:
   - Get all Users
   - Get a single user
#### How should this be manually tested?
- Clone repo
- Install dependencies. Run `npm install`
Add environment variables as specified in the env.sample file in a .env file
Run server `npm run dev:start`
Using Postman, make a `PATCH` request to 'localhost:9999/api/v1/users/'
You should receive a 200 status and all registered users
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#163356700
#### Screenshots (if appropriate)
#### Questions:
